### ISSUE TITLE
[Bug] Adding/removing Avatar in Profile and Workspace pages is not behaving  the same - reported by @Puneet-here

### ISSUE THREAD

**Issue by mvtglobally** [Expensify/App#9239]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Open settings > workspace > general setting
2. Tap on avatar and upload an image
3. Check the image will get saved itself
4. Now go back to settings page and click on profile
5. Now tap on avatar and upload an image
6. Check the image won't get saved itself

## Expected Result:
The behavior should be consistent

## Actual Result:
The behavior isn't consistent

## Workaround:
unknown 

## Platform:

Where is this issue occurring?

- Web
- iOS
- Android
- Desktop App
- Mobile Web

**Version Number:** 1.1.69-0
**Reproducible in staging?:** Y 
**Reproducible in production?:** Y 
**Email or phone of affected tester (no customers):** 
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation 

https://expensify.slack.com/files/U037WGYT199/F03CVP974EA/screen_recording_2022-04-27_at_4.28.55_am.mov

**Expensify/Expensify Issue URL:** 
**Issue reported by:** @Puneet-here
**Slack conversation:** https://expensify.slack.com/archives/C01GTK53T8Q/p1651014262709629

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by MonilBhavsar** [comment]
As discussed in the slack, we'll update the workflow considering https://github.com/Expensify/App/pull/8511. I'm going to work on this.

**Comment by MonilBhavsar** [comment]
Making it weekly and is on hold

**Comment by MonilBhavsar** [comment]
Still on hold

**Comment by MonilBhavsar** [comment]
Still on hold

**Comment by MonilBhavsar** [comment]
Still on hold

**Comment by MonilBhavsar** [comment]
Still on hold

**Comment by MonilBhavsar** [comment]
Still on hold

**Comment by MonilBhavsar** [comment]
Still on hold

**Comment by MonilBhavsar** [comment]
It's unblocked now. Will try to get back to this.

**Comment by MonilBhavsar** [comment]
Oops, will be back soon

**Comment by MonilBhavsar** [comment]
Doing next week

**Comment by MonilBhavsar** [comment]
Doing this week

**Comment by MonilBhavsar** [comment]
Got engaged in other priority bugs, will get back to this soon

**Comment by Beamanator** [comment]
It's verrrry possible this is not a bug anymore - @jasperhuangg and I recently refactored this code while adding the new API commands `UpdateUserAvatar` and `UpdateWorkspaceAvatar` - I'll check what the flows look like now

**Comment by Puneet-here** [comment]
Hi, can we settle this as it was a valid bug report.

cc: @MonilBhavsar  @johncschuster

**Comment by JmillsExpensify** [comment]
Bump on @Puneet-here's comment.

**Comment by johncschuster** [comment]
Sorry team, I was behind here. Let me get caught up and get it settled.

**Comment by johncschuster** [comment]
Job posting is here: 
https://www.upwork.com/jobs/~013e47108e87e1c6b7

@Puneet-here can you apply to it so I can pay it out?

**Comment by Puneet-here** [comment]
> @Puneet-here can you apply to it so I can pay it out?

Applied just now.

**Comment by Puneet-here** [comment]
Accepted. Thanks!

**Comment by johncschuster** [comment]
Okey dokey! I have sent payment to @Puneet-here, the contract is complete, and the post has been closed.

Thanks for your patience while I learned the process on this one, Puneet.
